nlmod/gwf/recharge.py

```diff
@@ -163,15 +200,308 @@ def ds_to_evt(gwf, ds, pname="evt", nseg=1, surface=None, depth=None, **kwargs):
+        the array is obtained from ds. The default is 'kv'.
+    thtr : float, str or array-like
+        thtr is the residual (irreducible) water content of the UZF cell. This residual
+        water is not available to plants and will not drain into underlying aquifer
+        cells. When passed as string, the array is obtained from ds. The default is
+        0.1.
+    thts : float, str or array-like
+        thts is the saturated water content of the UZF cell. The values for saturated
+        and residual water content should be set in a manner that is consistent with the
+        specific yield value specified in the Storage Package. The saturated water
+        content must be greater than the residual content. When passed as string, the
+        array is obtained from ds. The default is 0.3.
+    thti : float, str or array-like
+        thti is the initial water content of the UZF cell. The value must be greater
+        than or equal to the residual water content and less than or equal to the
+        saturated water content. When passed as string, the array is obtained from ds.
+        The default is 0.1.
+    eps : float, str or array-like
+        eps is the exponent used in the Brooks-Corey function. The Brooks-Corey function
+        is used by UZF to calculated hydraulic conductivity under partially saturated
+        conditions as a function of water content and the user-specified saturated
+        hydraulic conductivity. Values must be between 3.5 and 14.0. When passed as
+        string, the array is obtained from ds. The default is 3.5.
+    landflag : xr.DataArray, optional
+        A DataArray with integer values, set to one for land surface cells indicating
+        that boundary conditions can be applied and data can be specified in the PERIOD
+        block. A value of 0 specifies a non-land surface cell. Landflag is determined
+        from ds when it is None. The default is None.
+    finf :float, str or array-like
+        The applied infiltration rate of the UZF cell (:math:`LT^{-1}`). When passed as
+        string, the array is obtained from ds. The default is "recharge".
+    pet : float, str or array-like
+        The potential evapotranspiration rate of the UZF cell and specified GWF cell.
+        Evapotranspiration is first removed from the unsaturated zone and any remaining
+        potential evapotranspiration is applied to the saturated zone. only
+        used if SIMULATE_ET. When passed as string, the array is obtained from ds. The
+        default is "evaporation".
+    extdp : float, optional
+        Value that defines the evapotranspiration extinction depth of the UZF cell, in
+        meters below the top of the model. Set to 2.0 meter when None. The default is
+        None.
+    extwc : float, optional
+        The evapotranspiration extinction water content of the UZF cell. Only used if
+        SIMULATE_ET and UNSAT_ETWC. Set to thtr when None. The default is None.
```

Comment:
mention these should be set to True

Reply:
Maybe nitpicking, but I think this makes it more clear these values should be set in in this function?